frontend/src/query_builder/AccordianList.jsx

```diff
@@ -157,9 +165,28 @@ export default class AccordianList extends Component {
                                 }
                             </div>
                         : null }
+
+                        { searchable &&
+                            /* NOTE: much of this structure is here just to match strange stuff in 'List-item' below so things align properly */
+                            <div className="px1 pt1">
```

Comment:
@kdoh took your suggestion and separated out the relevant parts into their own component.  check it out and let me know if that's what you were thinking.